Package.swift

```diff
@@ -381,19 +381,12 @@ let package = Package(
     .target(
       name: "Syndication",
       dependencies: [
+        "Models",
```

Comment:
this now depends on `Models` to get access to `Episode.Sequence`, but really a bunch of stuff in `Syndication` should prob be split into its own module (prob in swift-html), and then we could have a light layer for transforming episode specific models into to RSS content.